packages/notifications-controller/src/services/types.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'eth-ens-namehash';
```

Comment:
We tend not to use these if we can avoid it because it means clients will need to have these too. That said, we already have type declarations for these libraries declared in `types/` in the project root, so is this file necessary?

Reply:
addressed